src/functions_framework/__init__.py

```diff
@@ -65,6 +66,22 @@ def __init__(
         self.data = data
 
 
+class _LoggingHandler(io.TextIOWrapper):
+    """Logging replacement for stdout and stderr in GCF Python 3.7."""
+
+    def __init__(self, level, stderr=sys.stderr):
+        io.TextIOWrapper.__init__(self, io.StringIO(), encoding=stderr.encoding)
+        self.level = level
+        self.stderr = stderr
+
+    def write(self, out):
+        payload = dict(severity=self.level, message=out.rstrip("\n"))
+        return self.stderr.write(json.dumps(payload) + "\n")
+
+    def flush(self):
+        pass
```

Comment:
```suggestion
```

I think this isn't necessary? The `TextIOWrapper` instance already has a `flush()` method, unless we're trying to prevent `flush()` from working?

```
>>> import io
>>> io.TextIOWrapper
<class '_io.TextIOWrapper'>
>>> x = io.TextIOWrapper(io.StringIO())
>>> x.flush()
```

Reply:
Yeah the flush() just tries to flush the buffer (which would be an unused io.StringIO anyway)